SSO password spray playbook (#36548)
demisto/content: Packs/Core/Playbooks/playbook-SSO_Password_Spray_README.md

This playbook is designed to handle the following alerts:

- SSO Password Spray Threat Detected
- SSO Password Spray Activity Observed
- SSO Password Spray Involving a Honey User

Playbook Stages:

Triage:

- The playbook checks the IP reputation and fetches the events related to the SSO login attempts.

Early Containment:

- The playbook checks if the IP is suspicious. If it is, the playbook suggests blocking the IP.

Investigation:

> - The playbook assess the risk score of the user who successfully logged in and examines the legitimacy of the user agent. It verifies if the user has MFA configured and analyzes the timestamps of the login attempts to detect patterns.

Reviewer: to detect potential malicious automated patterns.

Playbook Overview:

> This playbook is designed to address the following alerts:

Reviewer 2: ```suggestion
This playbook addresses the following alerts:
```

- SSO Password Spray Threat Detected
- SSO Password Spray Activity Observed
- SSO Password Spray Involving a Honey User

Playbook Stages:

Triage:

> - The playbook will check the IP reputation and will fetch the events related to the SSO login attempts.
> 
> Early Containment:
> 
> - The playbook will check if the IP is suspicious. If it is, the playbook will suggest blocking the IP.

Reviewer 2: ```suggestion
- The playbook checks the IP reputation and fetches the events related to the SSO login attempts.

Early Containment:

- The playbook checks if the IP is suspicious. If it is, the playbook suggests blocking the IP.
```

Investigation:

> - The playbook assess the risk score of the user who successfully logged in and examine the legitimacy of the user agent. It verifies if the user has MFA configured. And analyzes the timestamps of the login attempts to detect patterns.

Reviewer 2: ```suggestion
- The playbook assess the risk score of the user who successfully logged in and examines the legitimacy of the user agent. It verifies if the user has MFA configured and analyzes the timestamps of the login attempts to detect patterns.
```

Containment:

> - If there is a successful login attempt and the user's risk score is high, or if the user agent is detected as suspicious, or if the time intervals were automated, the playbook will clear the user's session. If the user doesn't have MFA, the playbook will recommend expiring the user's password.

Reviewer 2: ```suggestion
- If there is a successful login attempt and the user's risk score is high, or if the user agent is detected as suspicious, or if the time intervals were automated, the playbook clears the user's session. If the user doesn't have MFA, the playbook recommends expiring the user's password.
```

Requirements:

> For any response action, you will need one of the following integrations:

Reviewer 2: ```suggestion
For any response action, you need one of the following integrations:
```
